x-pack/test/pki_api_integration/apis/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function({ loadTestFile }: FtrProviderContext) {
+  describe('apis PKI', function() {
+    this.tags('ciGroup6');
```

Comment:
__note:__ ciGroup6 is big now, maybe we should move security tests to a separate new group at some point :man_shrugging:

Reply:
Spencer bot magically balances the ci groups for us, right? ;)

Reply:
Haha, you're right, at least it's supposed to work like that :)